examples/ffi/exporter.cpp

```diff
@@ -120,15 +120,9 @@ int main(int argc, char* argv[])
 
     Holder<ddprof_ffi_ProfileExporterV3> exporter{exporter_new_result.ok};
 
-    ddprof_ffi_Buffer profile_buffer = {
-        .ptr = encoded_profile->buffer.ptr,
-        .len = encoded_profile->buffer.len,
-        .capacity = encoded_profile->buffer.capacity,
-    };
-
     ddprof_ffi_File files_[] = {{
         .name = to_byteslice("auto.pprof"),
-        .file = &profile_buffer,
+        .file = (ddprof_ffi_ByteSlice) {.ptr = encoded_profile->buffer.ptr, .len = encoded_profile->buffer.len},
```

Comment:
[Minor] cast is not necessary here:
```suggestion
        .file = {.ptr = encoded_profile->buffer.ptr, .len = encoded_profile->buffer.len},
```